oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/source/DefaultScopeDefine.java

```diff
@@ -163,18 +168,18 @@ private static final void addNewScope(ScopeDeclaration declaration, Class origin
         int id = declaration.id();
         if (ID_2_NAME.containsKey(id)) {
             throw new UnexpectedException(
-                "ScopeDeclaration id=" + id + " at " + originalClass.getName() + " has conflict with another named " + ID_2_NAME
-                    .get(id));
+                    "ScopeDeclaration id=" + id + " at " + originalClass.getName() + " has conflict with another named " + ID_2_NAME
+                            .get(id));
```

Comment:
All following changes in this file from here, are not necessary. You seem to have different format settings compared with others. Maybe it still could pass format check, but these should not exist in the git changes.

Reply:
There is a setting file in the root. Have you imported it?

Reply:
I  don't know `setting file in the root` you said

Reply:
https://github.com/apache/skywalking/blob/master/codeStyle.xml This is for IntelliJ